TaskForces/a11y/reports/audio-playback/index.html

```diff
@@ -119,52 +122,57 @@ <h2>Full Audio Publications</h2>
 
 			<p>Some reading devices will omit the text, making it appear as though an EPUB is an audiobook, but, unlike
 				audiobooks, there is always a minimal amout of text that has to be provided. Publications with full
```

Comment:
a tiny misspelling: amout > amount

Reply:
Thanks for the reports! I've corrected these typos in the latest commit.